paasta_tools/check_chronos_jobs.py

```diff
@@ -42,8 +52,14 @@ def compose_monitoring_overrides_for_service(chronos_job_config, soa_dir):
     monitoring_overrides['check_every'] = '1m'
     monitoring_overrides['runbook'] = monitoring_tools.get_runbook(
         monitoring_overrides, chronos_job_config.service, soa_dir=soa_dir)
+
+    if 'realert_every' not in monitoring_overrides:
+        guessed_realert_every = _guess_realert_every(chronos_job_config)
+        if guessed_realert_every is not None:
+            monitoring_overrides['realert_every'] = guessed_realert_every
     monitoring_overrides['realert_every'] = monitoring_tools.get_realert_every(
         monitoring_overrides, chronos_job_config.service, soa_dir=soa_dir)
+
     return monitoring_overrides
 
 
```

Comment:
Can you clarify the alert text to let users know that the fact that they are getting the alert is not an indication that it tried to run, it is just a reminder that the last run status is failing. 

Bonus points if you just say in the alert text how often they should expect the reminder?